### PR TITLE
Multivariate

### PR DESCRIPTION
**Pull request by vviers**
early attempt at #16 

Looking fairly good but:

1. no error bars (or error squares)
1. view resets to original everytime we move the sliders, so you cannot decide on a view you like and then work using it

Have a good week end !

PS: some libraries to add to DESCRIPTION:
- plotly
- reshape2

### CODE REVIEW

**Comment by floswald** [comment]
awesome!
few things:
* can we have

**Comment by floswald** [comment]
this one is not quite there yet I think. 
* can one fix the box somehow? whenever one changes the sliders, the box changes slightly which is annoying
* we need a surface to be opaque=0.5 or so
* smaller points would be good

**Comment by vviers** [comment]
Hi ! What do you mean by `opacity`exactly? `opacity = .5` returns a very transparent surface (it's sort of the equivalent of `alpha` in ggplot)

**Comment by floswald** [comment]
* yes, alpha is what i meant.
* so for some reason the other version in `lm-example.Rmd` is much nicer
* i see it's hard to squash the box a bit. it should be less tall in my opinion.
* it's good otherwise.

File: inst/tutorials/chapter4/mutlivariate_standalone.Rmd [hunk 1]
```diff
@@ -90,23 +89,24 @@ output$reg_mul <- renderPlotly({
...
     yaxis = axy,
-    zaxis = axz)
+    zaxis = axz, 
+    aspectmode = "cube")
   
   if ((a == a_true) && (b1 == b1_true) && (b2 == b2_true)){
     plot_ly(x = possible_x1, y = possible_x2, z = surf_matrix)  %>%
-    add_surface(name = "Regression Plane", hoverinfo = 'name', opacity = .75) %>%
+    add_surface(name = "Regression Plane", hoverinfo = 'name', opacity = .5, surfacecolor = rep('green', length(possible_x1))) %>%
     hide_colorbar()%>%
```
Comment (floswald) [inline inst/tutorials/chapter4/mutlivariate_standalone.Rmd]: for some reason, when i get the correct estimates (green points), the z-coordinate is no longer fixed. it's perfect while the slider is wrong.
can you check that?

**Comment by vviers** [comment]
woops yes I didn't pay attention, thanks!

done! :smile:

*Vincent Viers*
SciencesPo Paris - Euro-American program / University of California -
Berkeley
(+33) 6 32 58 78 07
​ / +1 (510) 813 8098​
vincent.viers@sciencespo.fr
​ / vincent.viers@berkeley.edu​

**Pull request merged**